src/components/Contacts/ContactDetails/ContactDetails.tsx

```diff
@@ -0,0 +1,35 @@
+import { AppBar, Box, Tab, Tabs } from '@material-ui/core';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ContactDetailsHeader } from './ContactDetailsHeader/ContactDetailsHeader';
+
+interface Props {
+  accountListId: string;
+  contactId: string;
+  onClose: () => void;
```

Comment:
Is this ```onClose``` prop needed at the moment? It doesn't appear to be used in this component. Or will it be used in the future?

Reply:
Yeah, I will build out the ContactDetailsHeader in another PR.  I added this anticipating that it will be used soon -- there needs to be a way to close the details panel after it's opened.